tests/src/Functional/CommerceBrowserTestBaseTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Drupal\Tests\commerce\Functional;
+
+/**
+ * Tests CommerceBrowserTestBase functionality.
+ *
+ * @group commerce
+ */
+class CommerceBrowserTestBaseTest extends CommerceBrowserTestBase {
```

Comment:
All right, makes sense. Core did it too for base test, so I went "what the hell, why not"